server/database/PgMigrationRepositories.js

```diff
@@ -0,0 +1,56 @@
+const BaseRepository = require('./BaseRepository');
+
```

Comment:
We can remove this file from the project. Only relevant for field-data-service

Reply:
The CaptureRepository and EventRepository do extend the BaseRepository, and it seems to be a good base level for talking to the database so I left it.